content/ember/v1/non-standard-ways.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: non-standard-ways-of-calling-code-ember-set-code-and-code-ember-get-code
-title: Non-Standard Ways of Calling `Ember.set` and `Ember.get`
```

Comment:
re: https://embercommunity.slack.com/archives/C04KG57CF/p1519239524000073 

I guess maybe we want to clean up the CSS of this instead to display backtick correctly. Because the `id` has to keep `code` in it. Sorry, I found this confusing and was just discussing `id`'s and `title`'s on Slack...

Reply:
reverted here https://github.com/ember-learn/deprecation-app/pull/96